### PR TITLE
S1ft1 further styling

### PR DESCRIPTION
**Pull request by MotNosniktaw**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -10,29 +10,33 @@
     <title>Guitar Collection</title>
     <link rel="stylesheet" type="text/css" href="normalize-8-0-1.css">
     <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Barlow|Cairo|Catamaran|Maven+Pro|Rajdhani&display=swap"
```
Comment (mallowmew) [inline index.php]: You're only using Catamaran in the page styling - you can remove the unused fonts.

**Review by mallowmew — COMMENTED** [review]
You should delete one of your .sql databases

File: styles.css [hunk 2]
```diff
@@ -1,20 +1,199 @@
...
+    -webkit-box-sizing: border-box;
     box-sizing: border-box;
+    font-family: 'Catamaran', sans-serif;
 }
 
 body {
     font-size: 62.5%;
+    background-image: url("./img/34216.jpg");
+    background-attachment: fixed;
+    /*background-color: rgba(20, 20, 255, 1);*/
```
Comment (mallowmew) [inline styles.css]: Remove commented lines

File: styles.css [hunk 3]
```diff
@@ -1,20 +1,199 @@
...
+
+.guitarsContainer {
+    border: 1px gray solid;
+    background-color: rgba(240, 240, 255, 0.8);
+    width: 90%;
+    margin: 0 auto;
+    font-size: 1.6rem;
 }
 
 .row {
```
Comment (mallowmew) [inline styles.css]: None of these rules in .row seem to do anything

File: styles.css [hunk 4]
```diff
@@ -1,20 +1,199 @@
...
+.column1 {
+    width: 4%;
+}
+
+.column2 {
+    width: 25%;
+}
+
+.item .column2{
+    background-color: rgba(240, 240, 255, 0.8);
```
Comment (mallowmew) [inline styles.css]: Do you want to set these cells opacity different to the other cells?

File: index.php [hunk 5]
```diff
@@ -10,29 +10,33 @@
     <title>Guitar Collection</title>
     <link rel="stylesheet" type="text/css" href="normalize-8-0-1.css">
     <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Catamaran&display=swap"
```
Comment (sjr-mayden) [inline index.php:14]: Shouldn't this go above the link to your own CSS file?

File: index.php [hunk 6]
```diff
@@ -10,29 +10,33 @@
...
     <div class="row header">
-        <div class="item-detail">#</div>
-        <div class="item-detail">Image</div>
-        <div class="item-detail">Guitar</div>
-        <div class="item-detail">Year</div>
-        <div class="item-detail">Type</div>
-        <div class="item-detail">Country</div>
-        <div class="item-detail">Value</div>
-        <div class="item-detail">Date Acquired</div>
+        <div class="item-detail column1">#</div>
```
Comment (sjr-mayden) [inline index.php]: I'd give the new classes names, id-column, etc., rather than numbering them

File: index.php [hunk 7]
```diff
@@ -10,29 +10,33 @@
...
+        <div class="item-detail column2">Image</div>
+        <div class="item-detail column3">Guitar</div>
+        <div class="item-detail column4">Year</div>
+        <div class="item-detail column5">Type</div>
+        <div class="item-detail column6">Country</div>
+        <div class="item-detail column7">Value</div>
+        <div class="item-detail column8">Date Acquired</div>
     </div>
     <?php
     $db = getDB();
```
Comment (sjr-mayden) [inline index.php]: I'd move as much of the PHP to the top of the file as possible, getting the DB and preparing the guitars can be done up at the top and then the PHP inside the HTML can be kept to a minimum

File: styles.css [hunk 8]
```diff
@@ -1,20 +1,194 @@
...
+
+@media screen and (max-height: 600px) {
+    .item {
+        height: 35vh;
+    }
+
+    .guitarsContainer {
+        font-size: 1rem;
+    }
+
```
Comment (sjr-mayden) [inline styles.css]: Extra newlines in here

File: index.php [hunk 9]
```diff
@@ -10,29 +10,33 @@
...
-        <div class="item-detail">Country</div>
-        <div class="item-detail">Value</div>
-        <div class="item-detail">Date Acquired</div>
+        <div class="item-detail column1">#</div>
+        <div class="item-detail column2">Image</div>
+        <div class="item-detail column3">Guitar</div>
+        <div class="item-detail column4">Year</div>
+        <div class="item-detail column5">Type</div>
+        <div class="item-detail column6">Country</div>
+        <div class="item-detail column7">Value</div>
```
Comment (sjr-mayden) [inline index.php]: I suspect you don't want to output the values purely as integers, maybe concatenate a £ and a '.00' on the end

File: styles.css [hunk 10]
```diff
@@ -1,20 +1,194 @@
 * {
```
Comment (sjr-mayden) [inline styles.css]: I'd put this and the normalize file into a css folder of their own to keep the root of your project nice and tidy

File: styles.css (hunk 3 above)
Comment (sjr-mayden) [reply styles.css]: Agreed, the row class is always overridden

File: styles.css [hunk 11]
```diff
@@ -1,20 +1,194 @@
...
+    h1 {
+        font-size: 2rem;
+    }
+
+    p {
+        font-size: 1.4rem;
+    }
+}
+
+@media screen and (max-width: 1250px) {
```
Comment (sjr-mayden) [inline styles.css]: Screen width between 1000px and 900px wraps the text "Date Acquired" onto two lines and it overflows the border

Comment (sjr-mayden) [reply styles.css]: When the screen width is less than 900px the width of guitarsCollection is more than the total width of the columns

File: styles.css [hunk 12]
```diff
@@ -1,20 +1,194 @@
...
+        height: auto;
+    }
+}
+
+@media screen and (max-height: 600px) {
+    .item {
+        height: 35vh;
+    }
+
+    .guitarsContainer {
```
Comment (sjr-mayden) [inline styles.css]: CSS classes are named with hyphens instead of camelCase

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Review by sjr-mayden — APPROVED** [review]

**Pull request merged**